### ISSUE TITLE
Improve accuracy of runtime measurements

### ISSUE THREAD

**Issue by basimr** [project-lovelace/lovelace-engine#27]
Right now our runtime measurements may be inaccurate. Perhaps the runtime sum should be the sum over non-random test cases? Some thought needs to be put into this before improvements are made.

**Comment by ali-ramadhan** [comment]
Yeah it's kind of low priority (PL3?) as it's only for comparing code from different users.

**Comment by ali-ramadhan** [comment]
This issue is also complicated by the fact that we have to do it differently for every language :(
